server/app/views/components/LinkElement.java

```diff
@@ -119,10 +127,20 @@ public LinkElement setIcon(Icons icon) {
     return this;
   }
 
+  public LinkElement setIconPosition(IconPosition position) {
```

Comment:
How about adding icon position as second optional param of `setIcon`? To make sure that they both used together. Also

Reply:
great idea. done.